src/org/opendatakit/briefcase/export/Model.java

```diff
@@ -248,4 +257,57 @@ private long size() {
     }
     return children;
   }
+
+  public boolean isChoiceList() {
+    return Optional.ofNullable(controls.get(fqn()))
+        .map(control -> getDataType() == CHOICE_LIST || ControlType.from(control.getControlType()) == SELECT_MULTI)
+        .orElse(false);
+  }
+
+
+  public List<SelectChoice> getChoices() {
+    QuestionDef control = controls.get(fqn());
+    List<SelectChoice> staticChoices = Optional.ofNullable(control.getChoices()).orElse(emptyList());
+    return !staticChoices.isEmpty() ? staticChoices : control.getDynamicChoices().getChoices();
+  }
+
+  // TODO This should be defined in JavaRosa, like the DataType enum
```

Comment:
If we put this here now, do you want to create an issue in JR to move it later? Or move it now?

Reply:
I'd rather not be blocked by a JR change. Let's do it later.

I've created an issue and I've assigned it to me to not forget about this. https://github.com/opendatakit/javarosa/issues/351